tests/Drivers/OpenAITextTest.php

```diff
@@ -94,3 +96,51 @@
         "The game is at 3pm in Detroit. The weather there will be 75° and sunny. You probably won't need a coat."
     );
 });
+
+it('sends the organization header when set', function (): void {
+    config()->set('prism.providers.openai.organization', 'echolabs');
+
+    FixtureResponse::fakeResponseSequence('v1/chat/completions', 'openai/generate-text-with-a-prompt');
+
+    Prism::text()
+        ->using('openai', 'gpt-4')
+        ->withPrompt('Who are you?')();
+
+    Http::assertSent(fn (Request $request): bool => $request->header('OpenAI-Organization')[0] === 'echolabs');
+});
+
+it('does not sends the organization header', function (): void {
+    config()->offsetUnset('prism.providers.openai.organization');
+
+    FixtureResponse::fakeResponseSequence('v1/chat/completions', 'openai/generate-text-with-a-prompt');
+
+    Prism::text()
+        ->using('openai', 'gpt-4')
+        ->withPrompt('Who are you?')();
+
+    Http::assertSent(fn (Request $request): bool => empty($request->header('OpenAI-Organization')));
```

Comment:
I'd have to check, but this sees that the header is empty, but (see https://github.com/echolabsdev/prism/pull/10/files#r1795618106) I think we want to make sure the header isn't present at all.

Reply:
@sixlive , this works as expected. If the header is present, even if it doesn't have a value, then the tests will fail. This is 👍🏼